engine/sbom/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Viewing the SBOM for Docker images
+description: Viewing the Software Bill of Materials (SBOM) for Docker images
+keywords: Docker, sbon, Anchore, images, Syft, security
+toc_min: 1
+toc_max: 2
+---
+
+A Software Bill Of Materials (SBOM) is analogous to a packing list for a shipment; it’s all the components that make up the software, or were used to build it. For container images this includes the operating system packages that are installed (e.g.: ca-certificates) along with language specific packages that the software depends on (e.g.: log4j). The SBOM could include only some of this information or even more details, like the versions of components and where they came from.
+
+> :warning: *Note:* This command is considered _experimental_ and may change or be removed in the future.
+> Please leave feedback and report issues on [its repository](https://github.com/docker/sbom-cli-plugin).
+
+The experimental `docker sbom` command allows you to view the SBOM of a container image. Today, it does this today by scanning the layers of the image using the [Syft project](https://github.com/anchore/syft) but in future it may read the SBOM from the image itself or elsewhere.
```

Comment:
```suggestion
The experimental `docker sbom` command allows you to generate the SBOM of a container image. Today, it does this by scanning the layers of the image using the [Syft project](https://github.com/anchore/syft) but in future it may read the SBOM from the image itself or elsewhere.
```

A bit on the fence how much "in future" we should go into (if it's worth mentioning at all)

Reply:
Agree. Suggest we don't include forward-looking statements in Docs.

Reply:
Can we please add `{: target="_blank" rel="noopener" class="_"}` for external URLs so they open in a new tab? Thanks